src/Compiler/Service/ServiceParseTreeWalk.fs

```diff
@@ -260,12 +260,10 @@ module SyntaxTraversal =
             | [ x ] -> x ()
             | _ ->
 #if DEBUG
-                assert false
-                failwithf "multiple disjoint AST node ranges claimed to contain (%A) from %+A" pos debugObj
-#else
+                printf "multiple disjoint AST node ranges claimed to contain (%A) from %+A" pos debugObj
```

Comment:
`printfn` to make output less cluttered?

Reply:
I will do it (more) properly [here](url) - thanks for the tip though :)